RELEASING.md

```diff
@@ -12,76 +65,143 @@ export NEW_RELEASE=0.4.0.dev4
 poetry version $NEW_RELEASE
 ```
 
-## Update dependencies and run tests
+## Update dependencies
 
 ```bash
+poetry install --all-extras --sync
+```
+
+If you want to update all dependencies to latest:
```

Comment:
Perhaps we should do this always as part of the release process? This would make sure that the newest packages are then tested to work with the release build.

Reply:
It feels a bit late in the day, i.e. we've been testing a bunch of code and then just before we release we update all the dependencies.  I think that should probably be a an earlier explicit PR we do in the lead up to a release.

Reply:
I mean, it's a _prepare_ release PR after all and it just updates our own pins, so it should be fine.

Reply:
Done